.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,8 +2,8 @@ Fixes #
 
```

Comment:
@Shaurya-Developer No need to update this file. This is the template markdown for the pull request on Github!

That means this file sets the markdown for pull requests raised on the CircuitVerse repository.

Reply:
@vedant-jain03  I have made the required changes. Can you check it now?